web/profiles/custom/os2loop/modules/os2loop_documents/src/Helper/NodeHelper.php

```diff
@@ -123,7 +123,12 @@ public function preprocessNode(array &$variables) {
    * Implements hook_access().
    */
   public function nodeAccess(EntityInterface $entity, string $operation, AccountInterface $account) {
-    if ('delete' === $operation && $entity instanceof NodeInterface && self::CONTENT_TYPE_DOCUMENT === $entity->getType()) {
+    if (
+      0 < $account->id() &&
```

Comment:
Using https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Session%21AccountInterface.php/function/AccountInterface%3A%3AisAuthenticated/9.1.x here may be easier to understand.

I think we need a better check (hack!) for this since most users will be authenticated and we only want to show this message when actually trying to delete documents. Maybe we need to check the current route? Or does Drupal have a better way to handle this?